src/wallets/IdentityWallet.ts

```diff
@@ -341,6 +341,7 @@ export class IdentityWallet implements TLWallet {
       rawTx.gasPrice,
       rawTx.gasLimit
     )
+    rawTx.feeRecipient = rawTx.feeRecipient || metaTxFees.feeRecipient
     rawTx.currencyNetworkOfFees =
       rawTx.currencyNetworkOfFees || metaTxFees.currencyNetworkOfFees
```

Comment:
Same here, I though metaTxFee.currenctyNetworkOfFees can not be null

Reply:
`metaTxFee.currenctyNetworkOfFees` can be undefined. I thought that was what Daniel wanted to see.

Reply:
but in the fees opject. If it is undefined in rawTx, this is an invalid transaction.

Reply:
Ok, I did it. It makes it impossible now to derive the fees from the rawTx as they have different interpretation of the `currencyNetworkOfFees` and `feeRecipient` values, so the signer is responsible for giving them both to `Transaction`.

If the relay provider replies with 0 fees with no currency network and fee recipient, then `rawTx` will have `zeroAddress` for these and `txFees` will have `undefined`.